src/code/containers/fv-challenge-container.js

```diff
@@ -44,15 +45,18 @@ class FVChallengeContainer extends Component {
     const Template = templates[this.props.template],
           bgClasses = Template.backgroundClasses,
           scaleFactor = calcScaleFactor(containerWidth, containerHeight),
+          scaledLinesHeight = scaleFactor * bgLinesHeight,
           scaledTopHudHeight = scaleFactor * topHudHeight,
           scaledImageWidth = scaleFactor * bgImageWidth,
           scaledImageHeight = scaleFactor * bgImageHeight,
           scaledBottomHudHeight = scaleFactor * bottomHudHeight,
           challengeHeight = scaledImageHeight - (scaledTopHudHeight + scaledBottomHudHeight),
-          style = scaledImageHeight ? { height: scaledImageHeight } : {};
+          style = scaledImageHeight
+                    ? { height: scaledImageHeight,
+                        backgroundSize: `auto ${scaledLinesHeight}px, auto ${scaledLinesHeight}px, contain` }
+                    : {};
```

Comment:
To clarify, is `scaledImageHeight` ever falsy? I can't think of a case where the default `auto auto` style below is used.